library/src/main/assets/google_map.html

```diff
@@ -182,6 +181,34 @@
   polylines = {};
 }
 
+function addPolygon(points, id, strokeWidth, strokeColor, fillColor) {
+  var pointsArray = [];
+
+  for (var i = 0; i < points.length; i++) {
+    var point = points[i];
+    pointsArray.push(point);
+  }
+
+  strokeColor = formatColor(strokeColor);
+  fillColor = formatColor(fillColor);
+  var polygon = new google.maps.Polygon({
+    map: map,
+    paths: pointsArray,
```

Comment:
can you just use `points` here directly? it seems like `pointsArray` is identical